### PR TITLE
allow to pass build arguments to docker client

### PR DESCRIPTION
**Pull request by kalessin**
Needed for passing things like private credentials without hard coding them in Dockerfile.

### CODE REVIEW

File: shub/image/build.py [hunk 1]
```diff
@@ -41,19 +41,22 @@
...
 
-def build_cmd(target, version, skip_tests, no_cache, filename='Dockerfile'):
+def build_cmd(target, version, skip_tests, no_cache, build_arg, filename='Dockerfile'):
     config = load_shub_config()
     create_scrapinghub_yml_wizard(config, target=target, image=True)
     client = utils.get_docker_client()
     project_dir = utils.get_project_dir()
     image = config.get_image(target)
     image_name = utils.format_image_name(image, version)
+    build_args = dict(a.split('=') for a in build_arg)
```
Comment (vshlapakov) [inline shub/image/build.py:59]: In theory, there might be build args with values having "=" inside, let's split by the first character only?
```suggestion
    build_args = dict(a.split('=', 1) for a in build_arg)
```

**Review by vshlapakov — COMMENTED** [review]
LGTM, added a minor suggestion :+1:

File: shub/image/build.py (hunk 1 above)
Comment (kalessin) [reply shub/image/build.py:59]: you are right. I will fix.

Comment (kalessin) [reply shub/image/build.py]: Done.

**Review by Gallaecio — APPROVED** [review]
Nice!

**Review by vshlapakov — APPROVED** [review]

**Comment by vshlapakov** [comment]
Thanks @kalessin :+1:

**Pull request merged**